lib/osf-components/addon/components/file-actions-menu/template.hbs

```diff
@@ -110,9 +110,35 @@
                 </Button>
             {{/if}}
         {{/if}}
+        {{#if @item.currentUserCanDelete}}
+            {{#if @item.providerIsOsfstorage}}
+                {{#if @item.isBoaFile}}
+                    {{#if this.isBoaEnabled}}
```

Comment:
This is a lot of nested ifs. Probably fine, though might be better as a getter in the component ts.

Reply:
Good point! I will distill this down to a getter on the component